src/main/java/dan200/computercraft/core/terminal/TextBuffer.java

```diff
@@ -61,76 +27,34 @@ public int length()
 
     public String read()
```

Comment:
I think we can probably just delete `read()` entirely.

Reply:
Yeah. I think I was confusing usages in the new tests that I wrote for actual usages.. :D